js/src/lib/Search/index.js

```diff
@@ -10,6 +10,11 @@ import { algolia } from '../config';
 const equals = (arr1, arr2) =>
   arr1.length === arr2.length && arr1.reduce((a, b, i) => a && arr2[i], true);
 
+// package overview page
+// home page (/:lang/)
+const shouldFocus = path =>
+  path.includes("/packages") || path.split("/").length === 3;
```

Comment:
Will we always have a trailing slash? As far as I can test, we always link to the version with a trailing slash but if I manually type "https://yarnpkg.com/en" it accepts the URL but the feature wouldn't work.

Reply:
oh good point, let's change it to `<= 3`